tasks/vm.yml

```diff
@@ -26,6 +26,12 @@
     interface: "{{ item }}"
   with_items: "{{ interfaces }}"
 
+- name: Create secure boot template variables
+  include_tasks: prepare_secure_boot.yml
+  when:
+    - boot_firmware == "efi"
+    - libvirt_vm_ovmf_uefi_shell_iso_path is defined
```

Comment:
Not supported on Debian?

Reply:
The Debian packages don't contain the UEFI shell ISO which contains the `EnrollDefaultKeys.efi `binary. WE would either need to build this from source or obtain it by some other means.